_adr/81/index.adoc

```diff
@@ -19,10 +19,267 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
 This means that when a user creates a Kafka instance they get to specify "how big" the instance they're creating should be. 
 
+== Goals
+// Bulleted list of outcomes that this ADR, if accepted, should help achieve
+
+* Define how instance scaling is modelled conceptually and in the interfaces between the UI, fleet manager and fleet shard.
+* Avoid tying our hands to any particular unit of scaling.
+* Enable other profiles of the service in the future, without having to change the contracts between components.
+
+== Non-goals
+// Bulleted list of outcomes that this ADR is not trying to achieve.
+
+* Consistency of scaling across different infrastructures (both on the same cloud, and between different clouds).
+* Define how the fleet manager manages shard capacity.
+* Storage scaling, which already supported in the fleet manager and shard, and is orthogonal to the scaling of clusters under consideration here.
+
+== Current situation
+// Where are we now?
+
+The service does not offer any scaling options, except for storage, which is orthogonal to this problem.
+
+== Proposal
+// What is the decision being proposed
+
+=== Conceptual model
+Conceptually we define a _profile_, which is an ordered list of _sizes_ of instance. 
+
+In addition to being a composite of sizes, a _profile_ has an id and a set of parameters that is common across all the sizes in the profile.
+
+A _size_ has an id and a set of parameter values.
+Each parameter value associates value with one of the parameters defined in the parent profile.
+Each of the sizes in a profile MUST have the same set of function parameters (e.g. ingress bandwidth), but the values MAY differ.
+
+The ordering of the list MUST be defined by the ordering of the functional attributes that define the service. 
+
+For example, for the "Standard" profile, we might have the sizes:
+
+`small`:: Ingress 30MB/s, egress 30MB/s, 1000 partitions, 500 connections per instance, max 100 connections per second, 1MB max message size.
+`medium`:: Ingress 60MB/s, egress 60MB/s, 2000 partitions, 1000 connections per instance, max 100 connections per second, 1MB max message size.
+`large`:: Ingress 90MB/s, egress 90MB/s, 3000 partitions, 1500 connections per instance, max 100 connections per second, 1MB max message size.
+
+In this example, the functional attributes scale linearly, with the exception of connection rate and max message size.
+In general, linearity SHALL NOT be a requirement, even for those attributes which do scale up.
+For example, it would have been acceptable to define the `large` ingress as 100M/s, even though this is not 3× the `small` ingress, even when the other `large` functional parameters are 3× their `small` values.
+
+=== General principles
+
+* An individual fleet shard is required to provide only a subset of all profiles supported by the service globally. 
```

Comment:
That would mean the Fleet Manager needs to factor in the supported profiles when picking a cluster for placement. How can the fleet manager know which profiles are supported by the fleet shard? Or is this purely a matter of fleet isolation? In that case, do we really need profile when we already have instance type?